cmd/minishift/cmd/profile/profile_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package profile
+
+import (
+	"testing"
+
+	"github.com/minishift/minishift/pkg/testing/cli"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+)
+
+func Test_validate_args_exit_when_profilename_empty(t *testing.T) {
+	tmpMinishiftHomeDir := cli.SetupTmpMinishiftHome(t)
```

Comment:
I don't think your test needs that. A test should be as minimal as possible. You are just doing input validation, there is no dependence on the Minishift home dir.

Reply:
Yes that crossed my mind too, but the `cli.TearDown` function needs a `tmpMinishiftHomedir` as the first argument, and if the teardown is not called i get lots of error that i don't understand, passing an empty string seems to work, is it okay to pass empty string to `cli.TearDown`